qemu/target-i386/cpu.h

```diff
@@ -699,7 +699,7 @@ typedef enum {
 
 typedef struct SegmentCache {
     uint32_t selector;
-    target_ulong base;
+    uint64_t base;   /* handle 32 or 64 bit CPUs */
```

Comment:
oh, why do we need to fix this code from Qemu? it is better to leave this unchanged, while we still expose `base` to our API as `uint64_t` for our convenience?